pagopa/io-app
# chore(IT Wallet): [SIW-780] Update `ItwPrProximityQrCodeScreen` with proximity saga

### This PR depends on #5362 

## Short description
Include a summary of the changes.

## List of changes proposed in this pull request
- Update ItwPrProximityQrCodeScreen
- Add mocked mDL response

## How to test
Start the app. After wallet activation try to request a credential like the mDL. Go to the credential detail and tap on "Show QR Code". Using a verifier app Scan te QR Code to start proximity flow. The credential should be correctly verified.
